packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -187,6 +187,27 @@ class DescopeWc extends BaseDescopeWc {
     return null;
   }
 
+  async #handleFlowRestart() {
+    this.loggerWrapper.debug('Trying to restart the flow');
+    const prevCompVersion = await this.getComponentsVersion();
+    this.getConfig.reset();
+    const compVersion = await this.getComponentsVersion();
+
+    if (prevCompVersion === compVersion) {
+      this.loggerWrapper.debug(
+        'Components version was not changed, restarting flow',
+      );
+      this.flowState.update({
+        stepId: null,
+        executionId: null,
+      });
+    } else {
+      this.loggerWrapper.error(
+        'Components version mismatch, please reload the page',
```

Comment:
@nirgur we said we will have callback in that case, no?

Reply:
I remember we talked about adding a flag to reload the flow
Which callback we should have in this case?

Reply:
What happens if component version  has changed as well? How can the descoper react to that?
Right now we only console log it, but they should have a way to react so they can refresh the browser and do whatever they want in that case.

Reply:
If they have a way to handle that error, let's document it.

Reply:
We are also dispatching an error 
If you believe the SDK needs an additional change to support this case, let’s define the expected behavior